tests/vite-plugin-pwa.ts

```diff
@@ -0,0 +1,25 @@
+import { runInRepo } from '../utils'
+import { RunOptions } from '../types'
+
+export async function test(options: RunOptions) {
+	if (options.viteMajor < 4) {
+		// only latest version
+		return
+	}
+
+	await runInRepo({
+		...options,
+		repo: 'vite-pwa/vite-plugin-pwa',
+		branch: 'userquin/feat-add-tests',
+		overrides: {
+			svelte: 'latest',
+			react: 'latest',
+			preact: 'latest',
+			vue: 'latest',
+			'solid-js': 'latest',
+		},
```

Comment:
So i'd suggest you remove any override not needed and add overrides for plugins that are used in vite-plugin-pwa tests only.

Reply:
We use vue, react, preact, solid and svelte plugins, we have an example for each framework and we need to build each example 4 times for each build + browser test (the only exception is TypeScrypt example, only built 1 time).

I'll need some help, since plugin depndencies only configured in the corresponding package.json: do I need to include each plugin as dev dependecy in root package.json?